lib/isolate_runner.dart

```diff
@@ -61,8 +61,8 @@ class IsolateRunner implements Runner {
     isolate.setErrorsFatal(false);
     var pingChannel = SingleResponseChannel();
     isolate.ping(pingChannel.port);
-    var commandPort = await channel.result;
-    var result = IsolateRunner(isolate, commandPort);
+    var commandPort = await channel.result as SendPort;
+    final result = IsolateRunner(isolate, commandPort);
```

Comment:
No need to make this `final`, just keep it as `var`. 
(Style guide now says to be consistent in whether to use `final` or `var` for locals, and this package generally doesn't use `final`).